hack/.staticcheck_failures

```diff
@@ -117,12 +115,8 @@ pkg/volume/quobyte
 pkg/volume/rbd
 pkg/volume/scaleio
 pkg/volume/storageos
-pkg/volume/util
 pkg/volume/util/fsquota
```

Comment:
Why not fix pkg/volume/util/fsquota as well?

Reply:
Sure. I will fix the following as well : 

```
pkg/kubelet/dockershim/libdocker
pkg/volume/util/operationexecutor
pkg/volume/util/fsquota
```